src/uwtools/scheduler.py

```diff
@@ -34,15 +32,41 @@ class JobScheduler(collections.UserDict):
 
     _map = {}
```

Comment:
Should `_map` be an instance variable instead of a class variable? It is different for all objects, right?

Reply:
It's the same of all objects of that class.